src/api/emailTemplates.ts

```diff
@@ -20,20 +21,72 @@ const emailTemplates = {
         break;
 
       case "PackageReview":
-        email = {
-          To: [OSFs.find((osf) => osf.Title === requestData.osf)?.email || ""],
-          Subject: "An RPA Request has been updated and requires your review",
-          Body: `RPA Request # ${request.requestId} has been assigned to you for initial approval.
-          
-          Link to request: ${_spPageContextInfo.webAbsoluteUrl}/app/index.aspx#/Request/${request.requestId}`,
-        };
+        {
+          switch (request.newSubStage) {
+            case "OSFReview":
+              email = {
+                To: [
+                  OSFs.find((osf) => osf.Title === requestData.osf)?.email ||
+                    "",
+                ],
+                CC: [requestData.supervisor.EMail],
+                Subject: `OSF Action: RPA ${requestData.positionTitle} is pending OSF review.`,
+                Body: `This email has been generated to inform you that a Request for Personnel Action (RPA) for - ${requestData.positionTitle} - has been submitted and is pending the OSF review.
+
+              To action this request, follow the below link:
+              <a href="${_spPageContextInfo.webAbsoluteUrl}/app/index.aspx#/Request/${request.requestId}">${_spPageContextInfo.webAbsoluteUrl}/app/index.aspx#/Request/${request.requestId}</a>`,
+              };
+              break;
+
+            default:
+              break;
+          }
+        }
         break;
 
       default:
         break;
     }
     return email?.To ? email : undefined;
   },
+  reviewStageChangesMade: (
+    requestData: RPARequest,
+    dirtyFields: any,
```

Comment:
This isn't any, it is a Partial<RPARequest>
```suggestion
    dirtyFields: Partial<RPARequest>,
```

Reply:
I guess it is actually Partial<Readonly<FieldNamesMarkedBoolean<RPARequest>>> -- so you have to also import { FieldNamesMarkedBoolean } from "react-hook-form";